lib/google_indexing.rb

```diff
@@ -9,9 +10,11 @@ class GoogleIndexing
   attr_reader :service, :url
 
   def initialize(url)
-    return if api_key_empty?
+    @api_client = GoogleApiClient.instance
+    return unless api_client.authorization
 
     @service = API::IndexingService.new
+    @service.authorization = api_client.authorization
     @url = url
```

Comment:
Main change: Now we set the authorization at the service scope level. Prior was a global setting.